launch/model_endpoint.py

```diff
@@ -216,8 +218,30 @@ def get(self) -> EndpointResponse:
 class Endpoint:
     """An abstract class that represent any kind of endpoints in Scale Launch"""
 
-    def __init__(self, model_endpoint: ModelEndpoint):
+    def __init__(self, model_endpoint: ModelEndpoint, client):
         self.model_endpoint = model_endpoint
+        self.client = client
+
+    def _update_model_endpoint(self):
```

Comment:
should this say `_update_model_endpoint_view`? since `_update_model_endpoint` sounds like (to me at least) that we're updating the actual model endpoint, not the client's view of it